docs/blender_docs/scene_gltf2.rst

```diff
@@ -230,6 +232,52 @@ channels if they are not needed.
    the Principled BSDF node directly.
 
 
+Clearcoat
+^^^^^^^^^
+
+When the Clearcoat input on the Principled BSDF node has a non-zero default value or
+Image Texture node connected, the ``KHR_materials_clearcoat`` glTF extension will be
+included in the export. This extension will also include a value or Image Texture
+from the Clearcoat Roughness input if available.
+
+If Image Textures are used, glTF requires that the clearcoat values be written to
+the red (``R``) channel, and clearcoat roughness to the green (``G``) channel.
+If monochrome images are connected, the addon will remap them to these color channels
+during export.
+
+The Clearcoat Normal input accepts the same kinds of inputs as the base Normal input,
+specifically a tangent-space normal map with +Y up, and a used-defined strength.
+This input can re-use the same normal map that the base material is using, or can
+be assigned its own normal map, or can be left disconnected for a smooth coating.
+
+All clearcoat-related Image Texture node(s) should have their *Color Space* set to Non-Color.
+
+.. figure:: /images/addons_import-export_scene-gltf2_material-clearcoat.png
+
+   An example of a complex clearcoat application that will export correctly to glTF.
+   A much simpler, smooth coating could be applied from just the Principled BSDF node alone.
+
+
+Transmission
+^^^^^^^^^^^^
+
+When the Transmission input on the Principled BSDF node has a non-zero default value or
+Image Texture node connected, the ``KHR_materials_transmission`` glTF extension will be
+included in the export. When a texture is used, glTF stores the values in
+the red (``R``) channel. The *Color Space* should be set to Non-Color.
+
+Transmission is different from simple alpha blending, because transmission allows
+specular reflections, enabling materials such as glass. Typically the alpha blend mode
+should remain "Opaque," the default setting. glTF does not offer a separate
+"Transmission Roughness," but the material's base roughness can be used to blur the
+transmission.
+
+.. warning::
+
+   Transmission is complex for real-time rendering engines to implement, and support
+   for this extension is not yet widespread.
```

Comment:
Suggested additions:

"In real-time engines where transmission is supported, various technical limitations in the engine may determine which parts of the scene are visible through the transmissive surface. In particular, transmissive materials may not be visible behind other transmissive materials. These limitations affect physically-based transmission, but not simpler *Alpha Blend* transparency."

I'm also tempted to say something about thin surfaces and refraction, but couldn't come up with a straight-forward explanation that seemed appropriate here.

Reply:
I made a number of changes to the Transmission section.